tests/draft2019-09/anchor.json

```diff
@@ -2,9 +2,7 @@
     {
         "description": "Location-independent identifier",
         "schema": {
-            "allOf": [{
-                "$ref": "#foo"
-            }],
+            "$ref": "#foo",
```

Comment:
Not sure I follow why this is changing

Reply:
Wasn't "allOf" there because originally, "$ref" needed that and couldn't co-exist with other properties? I removed it because it's not necessary for the draft 2019-09 schema. If I were writing a test from scratch for this draft, I wouldn't use "allOf", and also, isn't it the same behaviour?

Reply:
Uh -- true -- I guess what I was thinking was "this makes it harder to see that this test is the same as ones in previous drafts", but fair enough about your point that let's just write the schema as it'd be written today, so retracted :)

Reply:
I'd prefer that tests in general don't depend on previous drafts, and are also written in that draft's style. Not only that, but these are still drafts, and future drafts shouldn't always be restricted by former drafts. That is, that's my personal preference and opinion. :)